js/trackers.js

```diff
@@ -20,8 +20,13 @@ settings.ready().then(() => {
 
 require.scopes.trackers = (function() {    
 
-    function isTracker(urlToCheck, currLocation, tabId, request) {
-        
+    function isTracker(urlToCheck, thisTab, request) {
+        let currLocation = thisTab.url
```

Comment:
You should assign a default here of empty string because on line 32 if there is some weird condition where that is undefined, its going to check `.match()` of undefined , and that will error out.
`let currLocation = thisTab.url || ''`